spec/fx/adapters/postgres/triggers_spec.rb

```diff
@@ -38,13 +38,12 @@
       expect(first.definition).to include("EXECUTE FUNCTION uppercase_users_name()")
 
       connection.execute "CREATE SCHEMA IF NOT EXISTS other;"
-      connection.execute "SET search_path = 'other';"
 
-      triggers = Fx::Adapters::Postgres::Triggers.new(connection).all
-
-      expect(triggers).to be_empty
+      connection.with schema_search_path: "other" do
```

Comment:
This change (and the corresponding one in `functions_spec.rb`) should also do away with the need to reset the search path `around :each` (edb9da0).

Reply:
Yeah the whole thing is strange to me, but the resetting at the beginning of each test is pretty OK for me as a solution.